tools/unix/translate_categories.sh

```diff
@@ -1,5 +1,17 @@
 #!/bin/bash
 
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+set -e -u
 
-${DIR}/translate.awk {en=en+ru+uk+de+fr+it+es+ko+ja+cs+nl+zh-TW+pl+pt+hu+th+zh-CN+ar+da+tr+sv} "$1"
+SRC=en
+WORD=$1
+if [[ "$#" -gt 2 ]]; then
```

Comment:
I prefer to always use double brackets - they are more powerful and there is a less chance for error. I can change this case though.